content/en/docs/contributing/release-process.md

```diff
@@ -155,16 +156,19 @@ page if a step is missing or if it is outdated.
 
 3. Update the release branch:
 
-      Update the release branch with the latest commits from the master
-      branch, as follows:
+   - **(initial alpha, subsequent alpha and initial beta)** The release branch
+      should already exist (it was created at the end of the last final
+      release). Update the release branch with the latest commits from the
+      master branch, as follows:
 
        ```bash
        # Must be run from the cert-manager repo folder.
        git fetch --all
        git branch --force release-1.0 origin/release-1.0
        git checkout release-1.0
-       git merge --ff-only origin/master # don't run for a point release!
+       git merge --ff-only origin/master
```

Comment:
question: was it intentional to remove this comment?

Reply:
Good catch, that is unintentional 😅

(I noticed that right above it says "initial alpha, subsequent alpha and initial beta" but I realize that it is good to remind people "only for point release" since this command should never be used for patch releases)